PublicationsRouter/v2/api/API.md

```diff
@@ -74,76 +74,94 @@ If you have publicly hosted content (e.g. splash pages, full-text web pages, or
         }
     ]
 
-### Validation Endpoint
+## Validation Endpoint
 
-The Validation API allows you to test that your data feed to the system will be successful.
+You must have **Publisher account** to access this endpoint. The Validation API allows you to test that your data feed to the system will be successful. 
 
-You must have "Publisher account" (as opposed to an "Institution repository account") to access this endpoint.
+NOTE: [Incoming Notification](https://github.com/sherpaservices/Public-Documentation/blob/master/PublicationsRouter/v2/api/IncomingNotification.md) structure.
 
-#### Metadata-only request
+### 1. Validate Metadata-only request
 
 If you are sending only the notification JSON, the request must take the form:
 
     POST /validate?api_key=<api_key>
-    Content-Type: application/json
+    Header:
+        Content-Type: application/json
+    Body: 
+        {Incoming Notification JSON}
     
-    [Incoming Notification JSON]
+### 2. Validate List of notifications with Metadata-only request
+
+If you are sending a list of notifications, the request must take the form:
+
+    POST /validate/list?api_key=<api_key>
+    Header:
+        Content-Type: application/json
+    Body:
+        # List of Incoming Notification JSON
+        [{"notification": Incoming, "id": 1}, {"notification": Incoming, "id": 2}, {"notification": Incoming, "id": 3}...]    
 
-#### Metadata + Package request
+NOTE: Make sure that an ID is sent for each Incoming notification as those IDs will be returned into a success or error list 
+
+### 3. Validate Metadata + Package request
 
 If you are sending binary content as well as the metadata, the request must take the form:
 
     POST /validate?api_key=<api_key>
-    Content-Type: multipart/form-data; boundary=FulltextBoundary
-    
-    --FulltextBoundary
-    
-    Content-Disposition: form-data; name="metadata"
-    Content-Type: application/json
-    
-    [Incoming Notification JSON]
-    
-    --FulltextBoundary
+    Header:
+        Content-Type: multipart/form-data; boundary=FulltextBoundary
     
-    Content-Disposition: form-data; name="content"
-    Content-Type: application/zip
-    
-    [Package]
-    
-    --FulltextBoundary--
+    Body:
+        --FulltextBoundary
+
+        Content-Disposition: form-data; name="metadata"
+        Content-Type: application/json
+
+        [Incoming Notification JSON]
```

Comment:
Just wondering if using square brackets here is confusing? I notice above that you used curly brackets...